src/main/webapp/admin_dashboard_script.js

```diff
@@ -1,231 +1,264 @@
+			var pos = {
+				lat: position.coords.latitude,
+				lng: position.coords.longitude
+			};
+
+			infoWindow.setPosition(pos);
+			infoWindow.setContent('This is your Neighborhood!');
+			infoWindow.open(map);
+			map.setCenter(pos);
+		}, function() {
+			handleLocationError(true, infoWindow, map.getCenter());
+		});
+	} else {
+		// Browser doesn't support Geolocation
+		handleLocationError(false, infoWindow, map.getCenter());
+	}
+}
+
+function handleLocationError(browserHasGeolocation, infoWindow, pos) {
+	infoWindow.setPosition(pos);
+	infoWindow.setContent(browserHasGeolocation ?
+		'Error: The Geolocation service failed.' :
+		'Error: Your browser doesn\'t support geolocation.');
+	infoWindow.open(map);
+}
+
+function load(file) {
+	var src = document.createElement("script");
+	src.setAttribute("type", "text/javascript");
+	src.setAttribute("src", file);
+	document.getElementsByTagName("head")[0].appendChild(src);
+}
+
+function getUserTasks() {
+	fetch("/admin-user-tasks")
+		.then((response) => response.json())
+		.then((tasks) => {
+			userTasksArray = tasks;
+			let taskSection = document.getElementById("user-tasks");
+			for (userTask of tasks) {
+				taskSection.innerHTML += addTask(userTask);
+			}
+		});
+}
+
+function getAdminTasks() {
+	fetch("/admin-tasks")
+		.then((response) => response.json())
+		.then((tasks) => {
+			adminTasksArray = tasks;
+			let taskSection = document.getElementById("admin-tasks");
+			for (adminTask of tasks) {
+				taskSection.innerHTML += addAdminTask(adminTask);
+            }
+		});
+}
+
+function addTask(task) {
+	let string = `<a href="#user-popup"><li class="admin-user-task"  onclick="openWithPopup('${task.keyString}')"><span><h3> ${task.category} </h3>`;
+	string += `<h4> ${task.owner} </h4></span> <a href="#"><span id="delete-btn" onclick=deleteTask('${task.keyString}')><i class="fas fa-trash fa-2x"></i></span></a></li></a>`;
+	return string;
+}
+
+function addAdminTask(task) {
+    let string = `<li class="admin-task" onclick="openWithAdmin('${task.keyString}')">`;
+    string += `<h3>${task.detail}</h3><h4>${task.owner}</h4>`;
+    string += `<div class="admin-task-infos"><p class="date"><i class="far fa-calendar"></i> ${task.date}</p>`;
+    string += `<p class="time"><i class="far fa-clock"></i> ${task.time}</p></div></li>`;
+    return string;
+}
+function searchTasks(id){
+	for(task of userTasksArray){
+		if(id == task.keyString){
+			return task;
+		}
+	}
+}
+
+function searchAdminTasks(id){
+	for(task of adminTasksArray){
+		if(id == task.keyString){
+			return task;
+		}
+	}
+}
+
+async function openWithPopup(id){
+	const task = await searchTasks(id);
+	document.getElementById("task-detail-input").value = task.detail;
+	document.getElementById("reward-input").value = task.reward;
+	document.getElementById("task-id-input").value = task.keyString;
+	document.getElementById("edit-category-input").value = task.category;
+}
+
+async function openWithAdmin(id){

```

Comment:
Would it be better to add a delete icon like on the user tasks so that we know that clicking on that means delete and not edit, like on the user tasks?